docs/routes.json

```diff
@@ -130,6 +130,10 @@
             "path": "/guides/schema-validation/syntax"
           }
         ]
+      },
+      {
```

Comment:
Can we please put this guide in the "Miscellaneous/Limits" section and include a link from the limits page for people who want to dive deeper into pricing specific limits?

<img width="277" alt="image" src="https://user-images.githubusercontent.com/2289736/220452711-f2e58ceb-0c0a-4ed5-86d4-1d79b3948999.png">